templates/index.html

```diff
@@ -1,8 +1,8 @@
 {% extends "base.html" %}
 <!--External links are differentiated on home page.-->
 {% block templatestyles %}
-a[href*="//"]:not([href*="{{config.base_url | safe}}"])::after {
-content: " ↗";
+<!-- a[href*="//"]:not([href*="{{config.base_url | safe}}"])::after {
```

Comment:
? This just styles external links on the homepage with the arrow. What's the rule on that right now, since a few links in the footer have it declared manually as well?

Reply:
Good Q - I want to hold off on arrows for now until we have clearer standards for them.